config/dev/local-images-template.yaml

```diff
@@ -22,6 +22,8 @@ spec:
           value: {{ .RegistryURL }}:5000/dpu-daemon:dev
         - name: IntelVspImage
           value: {{ .RegistryURL }}:5000/intel_vsp:dev
+        - name: MarvellVspImage
+          value: {{ .RegistryURL }}:5000/mrvl-vsp:dev
```

Comment:
If we want this to build and deploy automatically with our Makefile, this will need to match the env here: https://github.com/openshift/dpu-operator/blob/main/Makefile#L242

Reply:
I don't follow. Could you elaborate?

Reply:
Sorry disregard, I was looking at the wrong thing!